README.md

```diff
@@ -8,6 +8,97 @@ software below.
 
```

Comment:
Put the first paragraph (_"This is a fork [...]."_) after the title and _lomap_ description.